src/domain/domain.ts

```diff
@@ -23,8 +23,8 @@ export type DomainAggregate = Record<string, DomainAggregateItem | null>
 
 export type AddDomain = {
   name: string
-  target: AddDomainTarget
-  programType?: EntityType.Instance | EntityType.Program
+  target: AddDomainTarget | EntityType.Program | EntityType.Instance
```

Comment:
`AddDomainTarget` already contains `EntityType.Program | EntityType.Instance`, so this change is not needed